indico/modules/events/registration/models/forms.py

```diff
@@ -228,6 +229,19 @@ class RegistrationForm(db.Model):
         default=True
     )
 
+    ticket_template_id = db.Column(
```

Comment:
see the structure of the rest of the file: doc comment above the column, no empty line in between, and i would put the relationship right after the `event_new` one